contribs/gnohealth/main.go

```diff
@@ -4,11 +4,24 @@ import (
 	"context"
 	"os"
 
+	"github.com/gnolang/gno/contribs/gnohealth/internal/timestamp"
 	"github.com/gnolang/gno/tm2/pkg/commands"
 )
 
 func main() {
-	cmd := newHealthCmd(commands.NewDefaultIO())
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			ShortUsage: "gnohealth <subcommand> [flags]",
```

Comment:
```suggestion
			ShortUsage: "<subcommand> [flags]",
```

Likewise here (see other references for ShortUsage in the codebase)